DALK.PL_ANALYZER/Views/Matches/Index.cshtml

```diff
@@ -3,41 +3,77 @@
 @{
     ViewBag.Title = "Meczyki";
 }
+ 
+    @{
+        string allMatchesClass = "layer";
+        if (Model.Count == 1)
+        {
+            allMatchesClass += " singleMatch";
+        } else {
+            allMatchesClass += " matches";
+        }
+    }
+    <div class="@allMatchesClass">
+        @foreach (DALK.PL_ANALYZER.Models.Matches.Match m in Model)
```

Comment:
MVC does have a dedicated pattern for user defined classes. DisplayFor(..._) or EditorFor(...) thats a neat feature. 
ELSE :) Please create a dedicated view for PlayerMatch

Reply:
DONE.